openvdb/openvdb/tools/FastSweeping.h

```diff
@@ -930,12 +930,12 @@ struct FastSweeping<SdfGridT, ExtValueT>::MinMaxKernel
         for (auto leafIter = r.begin(); leafIter; ++leafIter) {
             for (auto voxelIter = leafIter->beginValueOn(); voxelIter; ++voxelIter) {
                 const SdfValueT v = *voxelIter;
-                const SdfValueT fltMinDif = std::abs(v + std::numeric_limits<SdfValueT>::max());
-                const SdfValueT fltMaxDif = std::abs(v - std::numeric_limits<SdfValueT>::max());
-                if (v < mMin && fltMinDif >= 1.0e-4f) mMin = v;
-                if (v > mMax && fltMaxDif >= 1.0e-4f) mMax = v;
-                if (fltMinDif < 1.0e-4f) mFltMinExists = true;
-                if (fltMaxDif < 1.0e-4f) mFltMaxExists = true;
+                const bool vEqFltMin = v == - std::numeric_limits<SdfValueT>::max();
```

Comment:
std::numeric_limits<SdfValueT>::lowest()?

Reply:
I think it should be `-std::numeric_limits<SdfValueT>::max()` because this file uses the convention of defining min/max that way. See, e.g. [line 911](https://github.com/AcademySoftwareFoundation/openvdb/blob/master/openvdb/openvdb/tools/FastSweeping.h#L911) or [line 1104](https://github.com/AcademySoftwareFoundation/openvdb/blob/master/openvdb/openvdb/tools/FastSweeping.h#L1104).